### ISSUE TITLE
Install Issues

### ISSUE THREAD

**Issue by swt83** [maebert/jrnl#449]
Ubuntu 14.04 LTS.  I'm not a python guy so I didn't have it installed.  I installed python, installed pip, installed jrnl, and I have all kinds of SSL errors.  I don't really want to spend 3 hours debugging the install process on this app.  Seems like your app requires specific dependencies, but you don't tell us what those are.

**Comment by notbalanced** [comment]
Hi,

I've installed this multiple times on Ubuntu and struggled with the dependencies as well. I finally wrote a cheat sheet that makes it work.

Hope it helps:
sudo apt-get install python-pip
sudo apt-get install python-dev
sudo apt-get install libffi-dev
sudo apt-get install libssl-dev
sudo pip install jrnl

I'm running the 2.o-rc1 branch version, so the dependencies may be a little different. YMMV.
